ga-ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json

```diff
@@ -233,6 +233,15 @@
     "Disabled": "F",
     "CaseAccessCategories": ""
   },
+  {
+    "RoleName" : "specifc-access-ctsc",
+    "Authorisation": "",
+    "CaseTypeID": "GENERALAPPLICATION",
+    "AccessProfiles": "caseworker-civil,ga-basic-access",
```

Comment:
do we need to give caseworker-civil?

Reply:
No not required for ctsc., added by mistake and removed caseworker-civil